docs/troubleshooting.md

```diff
@@ -0,0 +1,212 @@
+# Collector Troubleshooting
+
+This document describes common error conditions that Collector may encounter, and
+provides steps that can be taken to diagnose the cause, fix potential issues and
+return Collector to a working state.
+
+If the troubleshooting steps below do not solve the issue, please contact ACS
+support.
+
+## First Steps
+
+### Logs
+
+The first thing to look at are the logs from any failing Collectors. The troubleshooting
+guides below will offer examples of log messages you can expect to see for a number of
+error conditions, so having access to your own logs is invaluable.
+
+Depending on the environment, and accesses, you can get these in a number of ways.
+
+#### Kubectl
+
+Kubernetes is the easiest way to retrieve the logs, provided you have access to
+do so. 
+
+```sh
+$ kubectl get pods -n stackrox | grep collector
+collector-vclg5                      1/2     CrashLoopBackOff   2 (25s ago)   2m41s
+
+$ kubectl logs -n stackrox collector-vclg5 collector
```

Comment:
You might mention that in some cases they might want to do 

kubectl logs -n stackrox collector-vclg5 collector --previous